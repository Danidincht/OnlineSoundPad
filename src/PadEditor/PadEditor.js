import React from 'react';
import './PadEditor.css';

const PadEditor = () => (
	<div className="PadEditor">
		PadEditor Component
	</div>
);

export default PadEditor;
