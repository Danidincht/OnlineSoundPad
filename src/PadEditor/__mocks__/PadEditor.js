function PadEditor () {
	return (
		<div>PadEditorMockComponent</div>
	);
}

export default PadEditor;