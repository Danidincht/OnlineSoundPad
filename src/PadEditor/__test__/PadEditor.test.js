import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PadEditor from '../PadEditor';

describe('PadEditor', () => {
  it('it should render the PadEditor component', () => {
	// Given
	const component = (<PadEditor />);
    
	// When
	const page = render(component).baseElement;

	// Then
	expect(page).toMatchSnapshot();
  });
});