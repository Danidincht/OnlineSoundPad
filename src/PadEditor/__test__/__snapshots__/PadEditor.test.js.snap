// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PadEditor it should render the PadEditor component 1`] = `
<body>
  <div>
    <div
      class="PadEditor"
    >
      PadEditor Component
    </div>
  </div>
</body>
`;
